docs/topics/upgrade.md

```diff
@@ -0,0 +1,156 @@
+# Upgrading cluster
+
+Lokomotive cluster runs control plane viz. `apiserver`, `scheduler`, `controller-manager`, `coredns`, `kube-proxy` and `calico` as pods, just like any other workloads. To updrage them it is mostly about changing the image tag, which can be done using `kubectl`.
+
+**NOTE**: This document only covers upgrading from one patch version to another. So if you are on Kubernetes `1.15.1` then this guide explains how to upgrade to `1.15.5`. Since this is patch upgrade it is assumed that there were no major changes and just bug and security fixes.
```

Comment:
Should the doc title be Kubernetes patch release upgrades? And even the filename (instead of upgrades.md) 

So this doc https://github.com/kinvolk/lokomotive-kubernetes/blob/master/docs/topics/maintenance.md#upgrades can be linked for other upgrades and reference this one for patch releases?

Reply:
ok making such changes.